LayoutTests/fast/shadow-dom/shadow-style-invalidation-vw-units.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
Could this be a Web Platform Test?

Reply:
Sure, with a bit of modification to get rid of testRunner.waitUntilDone / notifyDone.

Reply:
```html
<!DOCTYPE html>
<html>
<body>
<script>

const iframe = document.createElement('iframe');
iframe.scrolling = 'no';
iframe.style = 'border: none; width: 50px; height: 50px; overflow: hidden';
iframe.src = `data:text/html,<!DOCTYPE html><style>head,html,body { padding: 0; margin: 0; }</style>
<div style="width: 100vw; height: 50vw; background: green"></div>
<div style="width: 100px; height: 100px; background: red;"><div id="host"></div></div>
<script>host.attachShadow({mode: 'closed'}).innerHTML =
    '<style> div { width: 100vw; height: 50vw; background: green; }</style><div></div>';
host.getBoundingClientRect();
</sc` + `ript>`;
iframe.onload = () => {
    iframe.style.width = '100px';
    iframe.style.height = '100px';
}
document.body.appendChild(iframe);

</script>
</body>
</html>
```